Jenkinsfile

```diff
@@ -0,0 +1,37 @@
+
+pipeline {
+    agent any
+
+        stages {
+
+            stage('Unittest') {
+                steps {
+                    /* use unittest */
+                    script {
+                       sh 'make test'
+                    }
+                }
+            }
+
+            stage('Build') {
+                steps {
+                    script {
+                     sh 'make build'   
+                    }
+                }
+            }
+
+        } // END stages
+
+    post {
+        success {
+            script {
+                echo 'slack me '
```

Comment:
I like to have slack notification even if it's working but it's a team decision here :) .